cloud/services/securitygroups/securitygroups.go

```diff
@@ -134,20 +175,31 @@ func ruleExists(rules []network.SecurityRule, rule network.SecurityRule) bool {
 	return false
 }
 
-func getRule(name, destinationPort string, priority int32) network.SecurityRule {
-	return network.SecurityRule{
-		Name: to.StringPtr(name),
+func getRule(ingress infrav1.IngressRule) network.SecurityRule {
```

Comment:
Thoughts on changing the name from `getRule` to `newIngressSecurityRule`?

Reply:
updated this bit in an amended commit.